andela/mrm-mobile
# #160141033 Display correct meeting room details

#### What does this PR do?
Adds functionality to display the correct meeting room data.

#### Description of Task to be completed?
- On clicking a meeting room button, show the actual room details.

#### How should this be manually tested?
- Run the application.
- Select a meeting room from a list of rooms; the room name is displayed correctly.
- Go ahead and select `Room info`; the correct details are shown.

#### Any background context you want to provide?
Prior to this PR, the default meeting room name in the layout file is what was being displayed.

#### What are the relevant pivotal tracker stories?
[#160141033](https://www.pivotaltracker.com/story/show/160141033)

#### Screenshots (if appropriate)
![ezgif com-video-to-gif 2](https://user-images.githubusercontent.com/7848682/44998233-8e3ee880-afbc-11e8-80ec-97735ca9514f.gif)
